### PR TITLE
fix: Elegant deletion of pods in sub-clusters (consistent with k8s)

### LINKED ISSUE #438: Forced deletion of the subcluster causes the TCP connection to be uninterrupted (the server returns FIN)
**What happened**:
Forced deletion of the subcluster causes the TCP connection to be uninterrupted (the server returns FIN)
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kosmos version:
- Others:

### PR DESCRIPTION
**Pull request by yuleichun-striving**
#### What type of PR is this?
/kind bug

#### Which issue(s) does this PR fix?

Fixes #438
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go [hunk 1]
```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/kosmos.io/kosmos/pkg/utils/podutils"
 )
 
+var gracePeriodSeconds *int64
```
Comment (duanmengkk) [inline pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go]: GracePeriodSeconds can be used directly as a local variable

**Comment by duanmengkk** [comment]
cherrypick

File: pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go (hunk 1 above)
Comment (yuleichun-striving) [reply pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go]: > GracePeriodSeconds can be used directly as a local variable

ok

**Comment by duanmengkk** [comment]
fix the ci

**Pull request closed without merging**